lib/index.js

```diff
@@ -104,7 +104,8 @@ function clean(obj) {
   for (var k in obj) {
     if (obj.hasOwnProperty(k)) {
       var value = obj[k];
-      if (value === null) continue;
+      // don't send null or undefined values
```

Comment:
The code is self-explanatory; isntead explain _why_ we're doing this. (e.g. `null and undefined properties cause Heap to reject the message, filter them out`